paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/LinkControllerPlaygroundActivity.kt

```diff
@@ -35,59 +32,25 @@ internal class LinkControllerPlaygroundActivity : AppCompatActivity() {
             return
         }
 
-        setContent {
-            val linkController = remember {
-                LinkController.create(
-                    activity = this,
-                    presentPaymentMethodsCallback = viewModel::onLinkControllerPresentPaymentMethod,
-                    lookupConsumerCallback = viewModel::onLinkControllerLookupConsumer,
-                    createPaymentMethodCallback = viewModel::onLinkControllerCreatePaymentMethod,
-                    authenticationCallback = viewModel::onLinkControllerAuthentication,
-                    registerConsumerCallback = viewModel::onRegisterConsumer,
-                )
-            }
-            LaunchedEffect(Unit) {
-                viewModel.onConfigureResult(
-                    result = linkController.configure(linkControllerConfig)
-                )
-            }
+        viewModel.onCreateActivity(this)
+        viewModel.configureLinkController(linkControllerConfig)
```

Comment:
Maybe this can be in. `viewModel.init` to avoid reconfiguring on config changes?

Reply:
For this playground activity, the config is passed through the Activity intent so we can't do it in the VM init block. I think in this toy example we can simply check the `configureResult` and no-op if already configured.

The "right" way to configure the controller will depend on the host app architecture.